### DISCUSSION TITLE
FR: support specifies the bufnrs for the buffer source

### DISCUSSION THREAD

**Discussion by gh-liu** [saghen/blink.cmp#410] [General]
I sometimes want words from other buffers in a tabpage, after I read the code, I find it always get current buffer, could we add a optional func to specifies the bufnrs?

**Comment by gh-liu** [comment]
[cmp-buffer](https://github.com/hrsh7th/cmp-buffer?tab=readme-ov-file#get_bufnrs-type-fun-number) support this.

**Comment by saghen** [comment]
Duplicate of #190

**Reply by gh-liu** [reply]
I am sorry to make this duplicate.
